LDM-652.tex

```diff
@@ -0,0 +1,210 @@
+\item Are the risks associated with the design of the LSST Science Platform understood and adequately captured? Are there any overlooked areas of risk?
+%
+% Note that in the event that  DM descope-10 is enacted prior to this review, this charge question will be reworded to be more specific,  acknowledging that a descope has occured. 
+% The question as to whether the remaining LSP is sufficient will be posed. 
+\item If cuts had to be made, what areas could be de-scoped with minimal implications for LSST science? What is the scope for use of third party-tooling in place of in-house development? 
+({\color{red}{Note: If DM-10 is invoked, or if the variant to freeze rather than eliminate the Portal is invoked, this question will be rephrased to reflect accordingly.}})
+
+%\color{red}{(Alternative if DM-10 is approved) In light of the recent implementation of de-scope item DM10: Eliminate the Portal Aspect of the Science Platform, are the the remaining two aspects of the LSP sufficient to 
+%ensure LSST science harvest covering the four key science themes? What, if any, third party-tooling could be substituted for the Portal aspect to ensure that the LSP can deliver the science promise of LSST?}
+\end{enumerate}
+
+In addition, the committee is asked to provide \newtext{actionable advice on addressing any issues raised during the course of the review as well as guidance based on experience that will ensure the success of the LSST Science Platform.}
+
+%%% The committee
+\section{Review Committee Composition}
+The review committee shall be composed of persons external to LSST Data Management
+and who are representative of the LSST DM stakeholders and principal users of the LSST Science Platform.
+
+% Note stakeholders comprise: LSST science collaborations, cross-subsystem integration, commissioning, early preparation by science collaborations
+
+%%% Committee Report
+\section{Committee report}
+At the conclusion of the review, the committee is requested to provide a written report  to the LSST Project Scientist within 2 weeks  detailing
+their major recommendations.  The report should contain the consensus of the committee's findings, comments and recommendations. The committee is also requested to
+provide a verbal exit briefing based around a closeout presentation on the last day to convey actions and comments. This closeout briefing need not
+necessarily be final and it is understood that the committee can revise their findings prior to presentation in the final written report.
+
+%%% Proposed agenda
+% Said we would do it here: March ? In the Spring. March/ early April. Before May IVOA meeting.
+\section{Proposed Agenda}
+We propose a full 2-day review, possibly with one full day and two half days either side in Spring 2019, possibly in March or April.
+The review committee are requested to be present for the full duration.
+
+(Draft) Proposed Review Agenda:
+
+\textbf{Wednesday 2019-02-20} \\
```

Comment:
Given that you say "March or April" above, perhaps the dates should be changed, or removed, or limited to weekdays?

Reply:
I will remove the dates